### ISSUE TITLE
[Bug] download_ckpt.py don't work properly

### ISSUE THREAD

**Issue by zydmtaichi** [One-2-3-45/One-2-3-45#56]
I run the download_ckpt.py script as instruction says and get unchanged error `Error downloading checkpoint: HTTP Error 403: Forbidden`.
It seems there is a wrong implement for getting the large weight files on huggingface repo via urllib. please check and fix.
all output pasted below:
![image](https://github.com/user-attachments/assets/0b42ee1f-259b-44f0-8e47-71379406f6c8)

**Comment by zydmtaichi** [comment]
@Dustinpro

**Comment by Dustinpro** [comment]
The link is valid as I just verified. Maybe you are in the region where Hugging Face is blocked.

**Comment by zydmtaichi** [comment]
> The link is valid as I just verified. Maybe you are in the region where Hugging Face is blocked.

yes, the link is valid, but missing header info in script cause the 403 problem.

I manually add the header and use `req = urllib.request.Request(url=xxx,header=xxx)` and replace the `url` to `req` in `urlopen`, this issue can solved.
